lib/data/external/AzureClient.js

```diff
@@ -53,6 +60,110 @@ class AzureClient {
         return this._translateMetaHeaders(metaHeaders);
     }
 
+    _putSinglePart(request, params, log, cb) {
```

Comment:
is this code different from what is in mystique? can these functions be added to arsenal?